src/View.Uno.WinUI/Extensions/PasswordBox/PasswordEnterCommand.MAUI.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if __ANDROID__ || __IOS__
```

Comment:
Since it's MAUI, add #if WINUI to engloble the whole class

Reply:
This file is in the WinUI project, there's no targets that aren't related to WinUI